src/SDKErrors.js

```diff
@@ -57,8 +57,10 @@ E('ERROR_UPDATE_EVENTMETADATA', '%s')
 E('ERROR_DELETE_ALL_EVENTMETADATA', '%s')
 E('ERROR_DELETE_EVENTMETADATA', '%s')
 E('ERROR_CREATE_REGISTRATION', '%s')
+E('ERROR_UPDATE_REGISTRATION', '%s')
 E('ERROR_GET_REGISTRATION', '%s')
 E('ERROR_GET_ALL_REGISTRATION', '%s')
+E('ERROR_GET_ALL_REGISTRATION_FOR_ORG', '%s')
```

Comment:
nitpick: change 'ERROR_GET_ALL_REGISTRATION_FOR_ORG' to 'ERROR_GET_ALL_REGISTRATIONS_FOR_ORG .
Same for 'ERROR_GET_ALL_REGISTRATION' above

Reply:
done